src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/UriExtensions.cs

```diff
@@ -14,13 +14,7 @@ public static string GetAbsoluteOrUNCPath(this Uri uri)
                 throw new ArgumentNullException(nameof(uri));
             }
 
-            if (uri.IsUnc)
-            {
-                // For UNC paths, AbsolutePath doesn't include the host name `//COMPUTERNAME/` part. So we need to use LocalPath instead.
-                return uri.LocalPath;
-            }
-
-            return uri.AbsolutePath;
+            return uri.LocalPath;
```

Comment:
How does this end up fixing it?

Reply:
AbsolutePath explodes when there's spaces in it for whatever reason I believe.

Reply:
It is because AbsolutePath is encoded (it contains %20 for spaces) and LocalPath isn't.